pkg/resources/dynamic_table.go

```diff
@@ -75,6 +75,34 @@ var dynamicTableShema = map[string]*schema.Schema{
 		Optional:    true,
 		Description: "Specifies a comment for the dynamic table.",
 	},
+	"refresh_mode": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "INCREMENTAL if the dynamic table will use incremental refreshes, or FULL if it will recompute the whole table on every refresh.",
+		ForceNew:    true,
+		DiffSuppressFunc: func(_, oldValue, newValue string, d *schema.ResourceData) bool {
```

Comment:
I think these two DiffSuppressFuncs are not needed. I think they can be removed even with the default specified.